docs/concepts/flows.md

```diff
@@ -608,10 +608,102 @@ Completed(message=None, type=COMPLETED, result='foo', flow_run_id=7240e6f5-f0a8-
 ```
 </div>
 
+## Pause a flow run
 
-## Flow run cancellation
+Prefect enables pausing an in-progress flow run for manual approval. Prefect exposes this functionality via the [`pause_flow_run`](/api-ref/prefect/engine/#prefect.engine.pause_flow_run) and [`resume_flow_run`](/api-ref/prefect/engine/#prefect.engine.resume_flow_run) functions, as well as via the Prefect Orion UI and Prefect Cloud. 
 
-Flow runs can be cancelled from the CLI, UI, REST API, or Python client. 
+Most simply, `pause_flow_run` can be called inside a flow. A timeout option can be supplied as well &mdash; after the specified number of seconds, the flow will fail if it hasn't been resumed.
+
+```python
+from prefect import task, flow, pause_flow_run, resume_flow_run
+
+@task
+async def marvin_setup():
+    return "a raft of ducks walk into a bar..."
+@task
+async def marvin_punchline():
+    return "it's a wonder none of them ducked!"
+@flow
+async def inspiring_joke():
+    await marvin_set()
+    await pause_flow_run(timeout=600)  # pauses for 10 minutes
+    await marvin_punchline()
+```
+
+Calling this flow will pause after the first task and wait for resumption.
+
+<div class="terminal">
+```bash
+await inspiring_joke()
+> "a raft of ducks walk into a bar..."
+```
+</div>
+
+Paused flows can be resumed via the resume flow run utility in another process.
+
+```python
+resume_flow_run(FLOW_RUN_ID)
+```
+
+The paused flow run will then finish!
+
+<div class="terminal">
+```
+> "it's a wonder none of them ducked!"
+```
+</div>
+
+Here is an example of a flow that stops executing in a paused state after one task, and will be rescheduled upon resuming. The stored result of the first task is retrieved instead of being rerun.
```

Comment:
```suggestion
Here is an example of a flow that does not block flow execution while paused. This flow will exit after one task, and will be rescheduled upon resuming. The stored result of the first task is retrieved instead of being rerun.
```